tools/jbrowse2/macros.xml

```diff
@@ -239,6 +240,7 @@
                 <option value="LinearPileupDisplay">LinearPileupDisplay - good for dense tracks at scale</option>
                 <option value="LinearArcDisplay">LinearArcDisplay</option>
             </param>
+            <param name="usebedscore" type="boolean" truevalue="true" falsevalue="false" checked="true" label="Use column 5 as a score for the display if bed format"/>            
```

Comment:
Would it be better to let the user select the column?

Maybe that is a stupid idea.
What happens when you disable that?

Reply:
Let's not go there yet. It's a minefield and just not worth exploring IMHO.
Bed format is too sinful for that.
This has a very specific use case at present although if useful it could be expanded, but unfortunately JB2 plugins are not trivial or flexible unfortunately.
Disabled should be default - will fix.
Enabled adds a new home made plugin to set the colour of the feature if score < 0 or > 0.